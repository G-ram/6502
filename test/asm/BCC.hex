38 4c 0a 06 a9 07 8d 00 01 00 90 f8 a9 08 8d 00
01
