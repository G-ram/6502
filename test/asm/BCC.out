A: 08
X: 00
Y: 00
P: 00010001
S: 00
PC: 611
100 : 08
101 : 00
